new/target_mode.md

```diff
@@ -0,0 +1,53 @@
+---
+RFC: unassigned
+Title: Chef Target Mode
+Author: Thom May <thom@chef.io>
+Status: Draft
+Type: Standards Track
+---
+
+# Chef Target Mode
+
+When deploying applications or systems, we frequently need to configure
+appliances where Chef cannot run natively. To allow us to do so, we need
+to run chef on a dedicated management node and interact with the
+appliance remotely, via an API.
+
+## Motivation
+
+    As a site reliability engineer,
+    I want to configure routers, switches and load balancers,
+    so that my application functions correctly.
+
+## Specification
+
+The concept of a node in chef is a 1:1 mapping with a client, enabling a
+single node to have several identities if needed. We can leverage this
+facility to create a target mode, allowing a management node to run chef
+for many appliances.
+Target mode is enabled by several new configuration options:
+ * `target`: The details for connectivity to the target
+   system. The result of this configuration option should be a
+   class instance that cookbooks can send messages to. This
+   enables target mode, disabling ohai entirely.
```

Comment:
It seems like a cookbook could already do most of this, and could expose its own API for setting a target.  The only thing that it really needs here is the disabling of default ohai plugins by default.  We could just build a boolean to do that easily.